jetson/db/file_transfer.py

```diff
@@ -0,0 +1,61 @@
+import paramiko
+import os
+import sys
+import config
+from contextlib import contextmanager
+
+
+@contextmanager
+def ftp_transfer():
+    """Sets up connection to target machine using SFTP and returns transfer function
+
+    Example Usage:
+        with ftp_transfer() as transfer:
+            transfer("./input_dir", "./output_dir", "file_name.ext")
+
+    Yields:
+        [function]: transfers file to output dir using SFTP
+
+        Args:
+            input_dir [string]: input directory for image, can be absolute or relative path
+            output_dir [string]: output directory for image on target machine, can be absolute or relative path
+            image_name [string]: name of file, include file type extension
+    """
+    transport = None
+    sftp = None
+    try:
+        transport = paramiko.Transport((config.FTPHOST, 22))
+        transport.connect(username=config.FTPUSER, password=config.FTPPASS)
+        sftp = paramiko.SFTPClient.from_transport(transport)
+        print('Sucessfully connected to host machine')
+
+        def transfer(input_dir, output_dir, image_name):
+            # ensure correct format for directory paths
+            if not output_dir.endswith('/'):
+                output_dir = output_dir + '/'
+
+            # transfer files from input directory to output directory
+            try:
+                # only transfer files that end with certain type(.jpg for testing purposes)
+                if input_dir.endswith('.py'):
+                    sftp.put(input_dir, output_dir + image_name)
+                    # remove the image from client side after successful transfer
+                    # os.remove(input_dir)
```

Comment:
is this code needed? if so uncomment it and if not please remove it

Reply:
I require it but for testing purposes, I'm leaving it commented.

Reply:
If you require it for testing purposes then please add a comment noting that. It was unclear initially and should be removed once testing is properly implemented.